src/components/HdModal.vue

```diff
@@ -134,6 +99,14 @@ export default {
       closeIcon,
     };
   },
+  computed: {
+    withIcon() {
+      return this.iconSrc.length > 0;
+    },
+    t() {
```

Comment:
I don't see this computed property used in the component. Maybe we could remove it??

Reply:
https://github.com/homeday-de/homeday-blocks/pull/732/commits/664aea4e6734685f1edf75779d57f89f88861898